Formula/portable-ruby@2.3.rb

```diff
@@ -0,0 +1,135 @@
+require File.expand_path("../../Abstract/portable-formula", __FILE__)
+
+class PortableRubyAT23 < PortableFormula
+  desc "Portable ruby 2.3"
+  homepage "https://www.ruby-lang.org/"
+  url "https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.4.tar.gz"
+  sha256 "98e18f17c933318d0e32fed3aea67e304f174d03170a38fd920c4fbe49fec0c3"
+
+  depends_on "make" => :build if OS.mac? && MacOS.version < :leopard
+  depends_on "makedepend" => :build
+  depends_on "pkg-config" => :build
+  depends_on "portable-readline" => :build
+  depends_on "portable-libyaml" => :build
+  depends_on "portable-openssl" => :build
+  if OS.linux?
+    depends_on "portable-ncurses" => :build
+    depends_on "portable-zlib" => :build
+  end
+
+  # Fixes the static build: https://bugs.ruby-lang.org/issues/13413
+  patch do
+    url "https://github.com/ruby/ruby/commit/b3dbeb6e90f316584f70e33f6bfb9d83fa5f30d3.patch?full_index=1"
```

Comment:
Yeah, this is a transitional feature for raw patch URLs that uses unabbreviated git refs instead of shortened ones.